src/components/Accordion/Accordion/__stories__/accordion.stories.mdx

```diff
@@ -0,0 +1,149 @@
+import Accordion from "../Accordion";
+import AccordionItem from "../../AccordionItem/AccordionItem";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../../storybook/functions/create-component-story";
+import Checkbox from "../../../Checkbox/Checkbox.js";
+import { WIZARD, BREADCRUBMS, STEPPER } from "../../../../storybook/components/related-components/component-description-map";
+import "./accordion.stories.scss";
+
+<Meta
+  title="Components/Accordion"
+  component={ Accordion }
+/>
+
+<!--- Component template -->
+
+export const accordionTemplate = (args) => {
+  return (
+    <div className="monday-storybook-accordion_small-wrapepr">
```

Comment:
please set the accordion width by using the className prop instead of by using a wrapper.
set the class prop inside the template and not by the templates args